crates/turborepo-lib/src/commands/unlink.rs

```diff
@@ -11,26 +11,48 @@ enum UnlinkSpacesResult {
 }
 
 fn unlink_remote_caching(base: &mut CommandBase) -> Result<()> {
-    base.delete_repo_config_file()
-        .context("could not unlink. Something went wrong")?;
+    let repo_config: &mut crate::config::RepoConfig = base.repo_config_mut()?;
+    let needs_disabling = repo_config.team_id().is_some() || repo_config.team_slug().is_some();
 
-    println!(
-        "{}",
-        base.ui.apply(GREY.apply_to("> Disabled Remote Caching"))
-    );
+    if needs_disabling {
+        base.repo_config_mut()?.set_team_id(None)?;
```

Comment:
Can we not reuse `repo_config` here?
```suggestion
        repo_config.set_team_id(None)?;
```

Reply:
I did this in isolation from the `*_config` handling change; this is a good catch.